test/components/legendTests.ts

```diff
@@ -413,4 +413,33 @@ describe("Legend", () => {
     legend.renderTo(svg);
     svg.remove();
   });
+
+  it("Title elements are created by default", () => {
+    color.domain(["foo", "bar", "baz"]);
+    legend.renderTo(svg);
+
+    let entries = (<any> legend)._element.selectAll(entrySelector);
+    let titles = entries.selectAll("title");
+    assert.strictEqual(titles[0].length, 1, "only one title node per legend entry should be present");
```

Comment:
`titles.size()` is [preferred](https://github.com/palantir/plottable/wiki/Testing-best-practices#checking-the-size-of-a-d3-selection).